src/EnergyPlus/BaseboardRadiator.cc

```diff
@@ -183,7 +183,7 @@ namespace BaseboardRadiator {
                                       DummyMdot,
                                       baseboard->Baseboard(BaseboardNum).WaterInletNode,
                                       baseboard->Baseboard(BaseboardNum).LoopNum,
-                                      baseboard->Baseboard(BaseboardNum).LoopSideNum,
+                                      baseboard->Baseboard(BaseboardNum).LoopSide,
```

Comment:
I renamed all `LoopSideNum` to `LoopSide` since it is an enum now. It made the PR quite large 😬  I can move these changes to another PR to make the review process easier, if necessary.

Reply:
I think you should revert this. This in particular change will get swallowed up by converting these quads to `baseboard->Baseboard(BaseboardNum).plantLoc` and there is no point in doing two large PRs that essentially cover the same code, especially if one of them is just a renaming.

Reply:
If you want to do a large renaming PR after this one, just do the `PlantLocation` PR instead.

Reply:
Got it. Will revert. Thanks for the quick reply!